lib/DateTime.php

```diff
@@ -153,10 +153,10 @@ public function setISODate($year, $week, $day = 1)
 		return parent::setISODate($year, $week, $day);
 	}
 
-	public function setTime($hour, $minute, $second = 0)
+	public function setTime($hour, $minute, $second = 0, $microseconds = 0)
```

Comment:
Are we sure that this will pass linting on PHP 7.0 too? If not, then we'll have to up the minimum version requirement in our `composer.json`, no?

Reply:
Not sure if it would be a linting error in terms of it being a syntax violation. If anything, it will give users a warning: https://3v4l.org/nJqn1

Don't think we'd _have_ to require 7.1 on a non-fatal.

Reply:
That would fatal error for us in development. 😞